docs/math.rst

```diff
@@ -3,22 +3,31 @@
 Mathematical Expressions
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-DALI allows you to use regular Python arithmetic operations and other mathematical functions in
-the :meth:`~nvidia.dali.Pipeline.define_graph` method on the values that are returned
-from invoking other operators.
+DALI allows you to use regular Python :ref:`arithmetic operations <Supported Arithmetic Operations>`
+and other :ref:`mathematical functions <Mathematical Functions>` in
+the pipeline definition (via :meth:`@pipeline_def <nvidia.dali.pipeline_def>` or within
+:meth:`~nvidia.dali.Pipeline.define_graph`) on the values that are returned from invoking
+other operators.
 
 The expressions that are used will be incorporated into the pipeline without needing to explicitly
-instantiate operators and will describe the element-wise operations on Tensors.
+instantiate operators and will describe the element-wise operations on tensors::
+
+    @pipeline_def
+    def my_pipeline():
+        """Create a pipeline which reads and decodes the images, scales channels by broadcasting
+        and clamps the result to [128, 255) range."""
+        img_files, _ = fn.readers.file(file_root="image_dir")
+        images = fn.decoders.image(img_files, device="mixed")
+        red_highlight = images * types.Constant(np.float32([1.25, 0.75, 0.75]))
```

Comment:
Would you mind stressing the `types.Constant` usage somewhere? That it is recommended to use arithmetic ops with numpy arrays wrapped into types.Constant to avoid ambiguity between numpy's and DALI's arithm ops?

I've seen a github issue with a code that (apart from other problems) was failing with the error caused by numpy's array as a left hand side of multiplication. And I was oblivious to the possible problems up until then too. :D

Reply:
![obraz](https://user-images.githubusercontent.com/8121971/214288806-e8764ed5-f118-477d-a74f-29731c9d5277.png)